server-spi/src/main/java/org/keycloak/models/UserModel.java

```diff
@@ -121,6 +121,22 @@
     
     void setCreatedTimestamp(Long timestamp);
 
+    /**
+     * Get timestamp of last user update. In case of new users, it will return the timestamp of creation.
+     *
+     * <p>May be null for old users last updated before this feature introduction.</p>
+     */
+    Long getLastUpdatedTimestamp();
+
+    void setLastUpdatedTimestamp(Long timestamp);
+
+    /**
+     * Marks this user as updated.
+     */
+    default void markAsUpdated() {
+        setLastUpdatedTimestamp(System.currentTimeMillis());
```

Comment:
I think you should use `Time.currentTimeMillis()` here.

Reply:
Thanks for the hint, I adjusted that.